packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.scss

```diff
@@ -1,5 +1,6 @@
 .wallets-dropdown {
     width: 100%;
+    max-width: 19.5rem;
```

Comment:
Why are we re-adding max-width to WalletDropdown, if we declare it here it'll change the other components which uses WalletDropdown. As per @wojciech-deriv's suggestion leaving the `width:100%` and declaring the max-width in Transactions.scss below should be enough to adjust the width right?

Reply:
Sorry, my mistake